### PR TITLE
Tiny fixes to README.

### PR DESCRIPTION
**Pull request by erikus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,6 @@ Options:
   -disable-keepalive    Disable keep-alive, prevents re-use of TCP
                         connections between different HTTP requests.
   -cpus                 Number of used cpu cores.
-                        (default for current machine is 8 cores)
   -host                 HTTP Host header.
   -http-trace         	Enable http trace detailed info on various events during request (Experimental)
```
Comment (rakyll) [inline README.md]: could you also replace this with -more. -http-trace is outdated, it is now called -more.

Comment (erikus) [inline README.md]: done

**Comment by erikus** [comment]
Done, I think.

**Comment by rakyll** [comment]
Thanks :)

**Pull request merged**